pkg/validation/port_values.go

```diff
@@ -0,0 +1,17 @@
+package validation
```

Comment:
add a license block

Reply:
so much for copying from `mime_types.go` ;)

Reply:
so much for copying from mime_types.go ;)

Reply:
😊